README.md

```diff
@@ -1 +1,33 @@
-# hershensh
\ No newline at end of file
+# hersh
+
+## Introduction
+This project aims to build a shell named `hersh`.
+The goal is for the shell to be fully compliant to [The Open Group Base Specifications, Issue 7, 2018 Edition](https://publications.opengroup.org/standards/unix/c181) which is identical to IEEE Std 1003.1, 2017 Edition.
+
+## Building
+### Requirements
+* C++ compiler supporting C++17.
+* [CMake](https://github.com/Kitware/CMake) version 3.8+.
+
+### Compiling
+
+Run the commands:
+
+```bash
+mkdir build
+cd build
+cmake ..
+make
+```
+
+This will create the `hersh` binary inside the `build` directory.
+
+## Usage
+Run the `hersh` binary located in the `build` directory to open the hersh shell.
```

Comment:
I think I don't get what this is aimed for. Will you change this once we have releases and won't expect people to build from source? Also shells aren't open, they're ran.

Reply:
It's meant to explain how to run the program.
I think building from source will always be an option, so this will always be relevant.
When there'll be releases, we can amend this section.